openmm_septop/septop_analysis.ipynb

```diff
@@ -0,0 +1,1065 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.            # Use standard deviation as error when more than 1 repeat
</pre><p>Maybe this is a more general problem with the error estimation but you would want to switch all edges to use the MBAR error if any edge has a single repeat; otherwise, the errors might have very different magnitudes due to how the MBAR estimator works and you would bias the MLE predictions to the edges with errors from MBAR. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenFreeEnergy/ExampleNotebooks/pull/225/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
No problem I can add that in!